app/ts/components/subcomponents/PreactBlocky.tsx

```diff
@@ -111,40 +109,39 @@ function generateIdenticon(options: BlockieProps, canvasRef: HTMLCanvasElement)
 	}
 
 	const opts = options || {}
-	const size = opts.size || 8
 	const scale = opts.scale || 4
-	const seed = opts.seed || Math.floor((Math.random()*Math.pow(10,16))).toString(16)
+	const seed = addressString(opts.address)
 
 	seedrand(seed)
 
 	const color = opts.color || createColor()
```

Comment:
What does this do if it doesn't affect the output blockie?

Reply:
hmm What's the question? Previously if the seed was missing, it generated a random seed

Reply:
If the user overrides the color, what happens?  The color of each pixel in the blockie needs to match exactly what is on Etherscan, MetaMask, etc. so I don't understand what setting this could do that wouldn't affect the validity of the blockie.

Reply:
oh, you are refering the `opts.color`? you are right it should be removed